config/testgrids/kubernetes/sig-docs/config.yaml

```diff
@@ -0,0 +1,7 @@
+dashboard_groups:
+- name: sig-docs
+  dashboard_names:
```

Comment:
Where will the job report the results? Can we forward the results to a link
accessible from the website/README?